FirelandsProject/cataclysm
# Core/Misc: Update locales_quest to simple system

### Changes proposed

- Update the quest_template_locale table, so that it has the structure of the current emulators.

### Issues addressed


- Closes

### SOURCE:



### Tests performed


- Windows 10
- In-game

### How to Test the Changes:


1. Add a quest to the table, and check if it is displayed in the language of the game client you are using.

### Known issues and TODO list



### Screenshot

![WoWScrnShot_073122_184353](https://user-images.githubusercontent.com/2810187/182046679-14bc4338-fda2-4997-b1d3-c74c1335e790.jpg)

---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
